client/src/main/java/org/apache/cloudstack/ServerDaemon.java

```diff
@@ -158,8 +164,8 @@ public void init(final DaemonContext context) {
     public void start() throws Exception {
         // Thread pool
         final QueuedThreadPool threadPool = new QueuedThreadPool();
-        threadPool.setMinThreads(10);
-        threadPool.setMaxThreads(500);
+        threadPool.setMinThreads(minThreads);
+        threadPool.setMaxThreads(maxThreads);
```

Comment:
@ahmedali6 I think, it's better to some restriction on min & max threads, as it's configurable now (also, need to add as a comment in the properties file if possible, check if any doc update is required)

Reply:
@sureshanaparti I added the threads config to the server. properties file. Do you have a suggestion about what  restrictions to do here?